sm/plugin_sm/src/plugin.rs

```diff
@@ -106,6 +106,7 @@ impl ExternalPluginCommand {
     pub fn check_module_type(&self, module: &SoftwareModule) -> Result<(), SoftwareError> {
         match &module.module_type {
             Some(name) if name == &self.name.clone() => Ok(()),
+            Some(name) if name == "default" => Ok(()),
```

Comment:
Use the const defined in the `plugin_manager`. Or defined a const here that is used in the `plugin_manager`.